### DISCUSSION TITLE
We encountered an unexpected error communicating with our servers.

### DISCUSSION THREAD

**Discussion by AzmiZouaghi** [cypress-io/cypress#27075] [Cypress Cloud]
For more than a week, we have a problem with the execution of tests with the record key, the test cases start to run and then stop. The message is:

We encountered an unexpected error talking about our servers.

We will retry 3 more times in 30 seconds...

The server's response was:

RequestError: Error: connect ETIMEDOUT [104.22.10.239:443](http://104.22.10.239:443/)

![image](https://github.com/cypress-io/cypress/assets/121963831/99652db4-6399-492a-8dc8-d6339b5c4412)

He repeats the execution then stops again.

Do you have any explanation please?

**Comment by arafatmusha** [comment]
I wish you good luck. Cypress is famous for this error. 
There are some requests that have been closed because of the term called **due to inactivity** . 
https://github.com/cypress-io/cypress/issues/24298
https://github.com/cypress-io/cypress/issues/26796
https://github.com/cypress-io/cypress/issues/3825
By the way, don't waste your time contacting their support. 
If the Cypress cloud is shared in your multiple teams, ask them what version of the Cypress they are using. Don't use the latest version of the Cypress. Cypress Cloud is known for not being capable of supporting the latest version of the Cypress. As of today, I updated our framework to Cypress 13, and it brings me this error again. So, I had to lower my version to Cypress 9, which has tons of vulnerabilities.
